test-dev/test_fuzzer_asy_invalid_samples.c

```diff
@@ -1,6 +1,5 @@
 #include "test.h"
 
-
```

Comment:
Two blank lines here is standard for all tests, except the ones where I placed comments here. I don't really know the reason for that, it predates my involvement with libxmp.

Reply:
Huh, never looked at it that way, i.e. existence of comments. Anyways, I'll keep the patch as it for now.